### ISSUE TITLE
CI docs about `devops` folder structure are inaccurate

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop#3704]
## Description

The folder structure described in https://docs.securedrop.org/en/latest/development/testing_continuous_integration.html does not match reality at https://github.com/freedomofpress/securedrop/tree/develop/devops . It's unclear to me whether other parts of the documentation are out of date. At minimum, we should update the docs to reflect current state.

**Comment by pierwill** [comment]
Went to see if there's a quick docs fix for this.

Looks like [Makefile](https://github.com/freedomofpress/securedrop/blob/develop/Makefile) might be stale? CI-related phony-targets of the form
```
.PHONY: ci-[thing
ci-[thing]
	./devops/scripts/ci-[thing].sh
```
appear to be pointing to scripts that've since been consilidated?

Then again, maybe I shouldn't be rooting around in Devops Land 😝

**Comment by redshiftzero** [comment]
These inaccurate docs were fixed by #3717, note these directories are named clearly enough that we don't need to explain again in docs (especially since these docs need to be maintained) so closing this
